project/styles/components/content/image.scss

```
@@ -2,6 +2,10 @@
   position: relative;
 }
 
+.image--isTextWidth {
```

we gebruiken kebab-case voor classnames

```suggestion
.image--is-text-width {
```